### ISSUE TITLE
Check pre-existing content for changes due to Duty Support

### ISSUE THREAD

**Issue by xiashtra** [quisquous/cactbot#5117]
Any triggers/timelines that existed prior to Duty Support coverage being added should be checked for changes.  Zones that have had Duty Support added so far:

Added in 6.1:
- [x] Sastasha *(no prior coverage)*
- [x] The Tam-Tara Deepcroft *(no prior coverage)*
- [x] Copperbell Mines *(no prior coverage)*
- [ ] The Bowl of Embers
- [x] The Thousand Maws of Toto-Rak *(no prior coverage)*
- [ ] Haukke Manor
- [ ] Brayflox's Longstop
- [ ] The Navel
- [ ] The Stone Vigil
- [x] The Howling Eye *(no prior coverage)*
- [x] Castrum Meridianum *(no prior coverage)*
- [x] The Praetorium *(no prior coverage)*
- [x] The Porta Decumana *(new zone)*

Added in 6.2:
- [ ] Snowcloak
- [x] The Keeper of the Lake *(no prior coverage)*
- [ ] Sohm Al
- [x] The Aery *(no prior coverage)*
- [ ] The Vault

**Comment by quisquous** [comment]
cc @JLGarber who mentioned something to me about Titan story mode.

**Comment by JLGarber** [comment]
Yes. I've been through most of this content so far and didn't notice any discrepancies besides Titan, but I can check through some of it during the long US holiday. My gut feeling is that most of it is fine with just a small number of actual changes. (Stone Vigil's second boss is completely different, for instance.)

**Comment by JLGarber** [comment]
Haukke Manor and Stone Vigil can be trivially marked as okay. They have no timelines, and the triggers all seem to be working.

**Comment by JLGarber** [comment]
The Vault seems fine as well. There was an actual change to Grinneaux, in that Faith Unmoving had an indicator added, but the action ID hasn't changed.

(It looks like I never added a trigger for the Holy Chains mechanic on Charibert though. Oops! I'll look at adding that this weekend.)

**Comment by xiashtra** [comment]
Sirensong Sea, no changes needed.
